src/models/Song.py

```diff
@@ -0,0 +1,24 @@
+class Song:
+	def __init__(self, apiSong):
```

Comment:
Nit: Your indenting has changed. Lets keep 2 spaces since python is indent based and 4 gets ugly fast